### ISSUE TITLE
Mempool stopped loading data after upgrading Bitcoin Core and Electrs 

### ISSUE THREAD

**Issue by alexandrz** [getumbrel/umbrel#1573]
Yesterday I upgraded Bitcoin Core and Electrs, but mempool stopped indexing new blocks. It now shows that the most recent block was mined 22 hours ago. Also, mempool information is only update don restart. I am running the most recent version of Umbrel, Bitcoin Core, Electrs, and Mempool. Here are some logs:

### sudo docker logs mempool_api_1

```
wait-for-it.sh: waiting 720 seconds for mariadb:3306
wait-for-it.sh: mariadb:3306 is available after 0 seconds
git not found, cannot parse git hash
Jan  7 10:01:32 [83] NOTICE: Starting Mempool Server... (?)
Jan  7 10:01:32 [83] INFO: Restoring mempool and blocks data from disk cache
Jan  7 10:01:32 [83] INFO: Database connection established.
Jan  7 10:01:32 [83] INFO: Starting statistics service
Jan  7 10:01:32 [83] INFO: Starting currency rates service
Jan  7 10:01:32 [83] INFO: Currency rates service will be queried over clearnet using https://price.bisq.wiz.biz/getAllMarketPrices
Jan  7 10:01:32 [83] NOTICE: Mempool Server is running on port 8999
Jan  7 10:01:33 [83] INFO: Updating latest mining pools from Github
Jan  7 10:01:33 [83] INFO: List of public pools will be queried over clearnet
Jan  7 10:01:38 [83] WARN: runMainLoop error: Loading block index…. Retrying in 10 sec.
Jan  7 10:01:42 [83] INFO: Connected to Electrum Server at 10.xx.xx.10:50001 (["electrs/0.9.10","1.4"])
Jan  7 10:01:48 [83] INFO: 171 blocks since tip. Fast forwarding to the 8 recent blocks
Jan  7 10:01:48 [83] INFO: Re-indexing skipped blocks and corresponding hashrates data
Jan  7 10:01:48 [83] ERR: Cannot fetch coinbase tx xxx. Reason: No such mempool transaction. Use -txindex or provide a block hash to enable blockchain transaction queries. Use gettransaction for wallet transactions.
Jan  7 10:01:48 [83] WARN: runMainLoop error: Cannot fetch coinbase tx xxx. Reason: No such mempool transaction. Use -txindex or provide a block hash to enable blockchain transaction queries. Use gettransaction for wallet transactions.. Retrying in 20 sec.
Jan  7 10:02:08 [83] INFO: 171 blocks since tip. Fast forwarding to the 8 recent blocks
Jan  7 10:02:08 [83] INFO: Re-indexing skipped blocks and corresponding hashrates data
```

### sudo docker logs mempool_mariadb_1

```
2023-01-07 10:01:29+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.5.12+maria~focal started.
2023-01-07 10:01:30 0 [Note] mysqld (mysqld 10.5.12-MariaDB-1:10.5.12+maria~focal) starting as process 1 ...
2023-01-07 10:01:30 0 [Note] InnoDB: Uses event mutexes
2023-01-07 10:01:30 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2023-01-07 10:01:30 0 [Note] InnoDB: Number of pools: 1
2023-01-07 10:01:30 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
2023-01-07 10:01:30 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2023-01-07 10:01:30 0 [Note] InnoDB: Using Linux native AIO
2023-01-07 10:01:30 0 [Note] InnoDB: Initializing buffer pool, total size = 134217728, chunk size = 134217728
2023-01-07 10:01:30 0 [Note] InnoDB: Completed initialization of buffer pool
2023-01-07 10:01:30 0 [Note] InnoDB: 128 rollback segments are active.
2023-01-07 10:01:30 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2023-01-07 10:01:30 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2023-01-07 10:01:30 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2023-01-07 10:01:30 0 [Note] InnoDB: 10.5.12 started; log sequence number 120922590; transaction id 433825
2023-01-07 10:01:30 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2023-01-07 10:01:30 0 [Note] Plugin 'FEEDBACK' is disabled.
2023-01-07 10:01:30 0 [Note] Server socket created on IP: '::'.
2023-01-07 10:01:30 0 [Warning] 'proxies_priv' entry '@% root@123d98fdec07' ignored in --skip-name-resolve mode.
2023-01-07 10:01:30 0 [Note] Reading of all Master_info entries succeeded
2023-01-07 10:01:30 0 [Note] Added new Master_info '' to hash table
2023-01-07 10:01:30 0 [Note] mysqld: ready for connections.
Version: '10.5.12-MariaDB-1:10.5.12+maria~focal'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
2023-01-07 10:01:30 0 [Note] InnoDB: Buffer pool(s) load completed at 230107 10:01:30
2023-01-07 10:01:30 3 [Warning] Aborted connection 3 to db: 'unconnected' user: 'unauthenticated' host: '10.xx.xx.26' (This connection closed normally without authentication)
2023-01-07 10:01:31 4 [Warning] Aborted connection 4 to db: 'unconnected' user: 'unauthenticated' host: '10.xx.xx.27' (This connection closed normally without authentication)
```

**Comment by alexandrz** [comment]
Re-installed mempool, but the issue is still there. It's not loading new blocks and mempool information.

<img width="1174" alt="Screenshot 2023-01-08 at 11 11 03" src="https://user-images.githubusercontent.com/81225/211190552-f802cb5b-897c-4082-a614-d74884cec25d.png">

**Comment by alexandrz** [comment]
It only loads mempool data on start.

**Comment by Syturio** [comment]
Check this out: https://community.getumbrel.com/t/mempool-broken-after-update/10824/13

**Comment by Syturio** [comment]
TL;DR

Go to advanced settings (accessed via the the three dots … in upper right corner of the app) and click ‘save and restart bitcoin node’.

**Comment by alexandrz** [comment]
@Syturio It works now. Thank you!
